### PR TITLE
libcircle fix for CCE

### PR DESCRIPTION
**Pull request by lukebroskop**
Add a libtool build dependency to libcircle when building w/ CCE

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
LGTM

**Comment by haampie** [comment]
Why are these not marked build dependencies?

And why do they depend on the choice of compiler?

**Comment by lukebroskop** [comment]
They are marked as build dependencies now. I had to make changes to the configure.ac file to compile w/ CCE.

File: var/spack/repos/builtin/packages/libcircle/package.py [hunk 1]
```diff
@@ -21,8 +21,9 @@ class Libcircle(AutotoolsPackage):
     depends_on('mpi')
     depends_on('pkgconfig', type='build')
     depends_on('libpciaccess', type='link')
-    depends_on('autoconf', when='%cce')
-    depends_on('automake', when='%cce')
+    depends_on('autoconf', when='%cce', type='build')
+    depends_on('automake', when='%cce', type='build')
+    depends_on('libtool', when='%cce' type='build')
```
Comment (haampie) [inline var/spack/repos/builtin/packages/libcircle/package.py]:
```suggestion
    depends_on('libtool', when='%cce', type='build')
```

**Review by haampie — CHANGES_REQUESTED** [review]

**Review by haampie — APPROVED** [review]

**Pull request merged**